mc-sema/cfgToLLVM/x86Instrs_Misc.cpp

```diff
@@ -709,6 +709,141 @@ static InstTransResult translate_LEA64_32r(NativeModulePtr natM, BasicBlock *&bl
     return doLeaRef<32>(natM, block, ip, inst);
 }
 
+static InstTransResult translate_CPUID32(NativeModulePtr natM, BasicBlock *&block, InstPtr ip, MCInst &inst) {
```

Comment:
Since I'd have to convert it to LLVM formatted inline assembly and supply all the arguments. There would also be an inline assembly call thats hard for analysis tools to reason about.

The current version is very far from perfect, but should give consistent results.